mne/minimum_norm/DeFleCT.py

```diff
@@ -0,0 +1,463 @@
+# Authors: Olaf Hauk <olaf.hauk@mrc-cbu.cam.ac.uk>
+#
+# License: BSD (3-clause)
+
+from copy import deepcopy
+import numpy as np
+import scipy
+from mne.utils import logger, verbose
+from mne import pick_types, pick_channels_forward, pick_channels_cov, SourceEstimate
+from mne.cov import prepare_noise_cov 
+from mne.cov import regularize as cov_regularize
+from mne.proj import make_eeg_average_ref_proj 
+from mne.minimum_norm.inverse import _prepare_forward
+
+
+def DeFleCT_matrix(F, P, i, t, lambda2_S=1/9.):
```

Comment:
private functions should start with underscore. Maybe also just make it lower case: `_deflect_matrix`